templates/internet-of-things/features/recovery.html

```diff
@@ -0,0 +1,137 @@
+          alt="",
+          width="400",
+          height="212",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+    </div>
+  </section>
+  
+  <section class="p-strip">
+    <div class="row u-sv3">
+      <h2>Ubuntu Core上设备恢复</h2>
+    </div>
+    <div class="row u-vertically-center">
+      <div class="p-card col-6 u-vertically-center">
+        <h3 class="p-card__title">ARM和x86</h3>
+        <p class="p-card__content">Ubuntu Core在ARM和x86 SoC上提供了一个稳健的设备恢复系统。设备可通过手动或者REST API远程恢复。</p>
+      </div>
+      <div class="p-card col-6 u-vertically-center">
+        <h3 class="p-card__title">预认证的开发板免费</h3>
+        <p class="p-card__content">恢复已在<a class="p-link--external" href="https://certification.ubuntu.com/iot">认证的设备</a>上可用，比如树莓派，无需额外成本。在未认证的开发板上，完整的认证Ubuntu Core需要适配费用。</p>
+      </div>
+    </div>
+  </section>
+  
+  <section class="p-strip--light">
+    <div class="row u-sv3">
+      <h2>如何工作</h2>
+      <p class="p-heading--4">Ubuntu Core 20引入了一个安全的恢复系统。</p>
+    </div>
+    <div class="row u-equal-height">
+      <div class="col-8">
+        <h3>恢复模式</h3>
+        <p>Ubuntu Core提供了一个在启动时可激活或远程API调用的恢复模式。在恢复模式中，Ubuntu Core提供了一个为用户进入恢复选项的用户界面。然后，操作系统将继续获取配置文件和预置了配置文件关联的snap应用。</p>
+      </div>
+      <div class="col-4 u-vertically-center u-align--center u-hide--small">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/f880a3bd-Enterprise+support.svg",
+          alt="",
+          width="150",
+          height="150",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+    </div>
+    <div class="u-fixed-width">
+      <hr class="p-separator" />
+    </div>
+    <div class="row u-equal-height">
+      <div class="col-8">
+        <h3>快照</h3>
+        <p>在恢复系统中可以备份同一设备的多个快照（snapshot）。这些快照反映了设置配置和系统上已安装的snap应用的集合。设备操作员可以在恢复系统中创建，命名和保存此类快照，这样可以在需要时快速恢复所需的系统状态。可以通过专用UI手动进行恢复，也可以通过API调用远程地进行。</p>
+      </div>
+      <div class="col-4 u-vertically-center u-align--center u-hide--small">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/7de21966-snapshot.svg",
+          alt="",
+          width="150",
+          height="150",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+    </div>
+  </section>
+  
+  <section class="p-strip is-deep">
+    <div class="row u-equal-height">
+      <div class="col-5 u-align--center u-hide--small">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg",
+              alt="",
+              width="281",
+              height="200",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+      </div>
+      <div class="col-7">
+        <h2>低接触设备恢复</h2>
+        <p>与Ubuntu安全专家联系，讨论您应用程序的高级安全要求。</p>
+        <p>
+          <a href="/contact" class="p-button--positive js-invoke-modal">Get in touch</a>

```

Comment:
This should be translated. If this section is the same on all pages you could copy the section into a partial file and include there with:
```
{% include "shared/_contact-strip.html" %}
```
_Files that begin with `_` indicates its a partial file._

Reply:
Hi Anthony, currently we don't have enough pages for different forms based on pages, so I remove the shared form quote. That's the next step of the plan. Thanks a lot.